Makefile

```diff
@@ -269,11 +269,11 @@ setup:
 	python3 extract_baserom.py
 	python3 extract_assets.py -j$(N_THREADS)
 
-test: $(ROM)
+run: $(ROM)
```

Comment:
```suggestion
run: $(ROM)
ifeq ($(EMULATOR),)
	$(error Emulator executable not set. Either change EMULATOR in the Makefile or define it as an environment variable to point to an emulator executable.)
endif
```
It would be nice to give a clear error if `EMULATOR` was not set to anything. I'm not happy about how the suggestion reads but it shows how to do it

Reply:
"Emulator not found. Set EMULATOR in the Makefile or define it as an environment variable."
short and sweet, gets the same info across?

Reply:
"Emulator path not set. Set EMULATOR in the Makefile or define it as an environment variable."

since this error doesn't cover the case where a path is set but is invalid

Reply:
done. i also tested to make sure it works. had to remove the period at the end since apparently "error" adds one on its own
![image](https://github.com/zeldaret/oot/assets/22045047/afbbdae3-637c-44c0-8dd0-75947e915a49)